### DISCUSSION TITLE
[Feature Request]: 增加对媒体库文件的管理

### DISCUSSION THREAD

**Discussion by insomnia417** [jxxghp/MoviePilot#379] [建议]
### 当前程序版本

1.0

### 功能改进

重开项目以后,我看到在对arr进行更多的支持,是否有考虑像sonarr/radarr一样,增加对媒体库的整体管理,甚至也可以照搬(想radarr/sonarr一样封面展示,可以进行文件的导入/删除的联动等等),以moviepilot为入口点,进行媒体库的增减删除管理,下载一体化

并且目前发现,mp在订阅方面,无法像sonarr一样很好的对单集或者整季进行订阅后展示和管理(个人认为很重要,希望改进)

为何提出此请求的原因:sonarr只支持tvdb,但是在国内环境,比如一些动漫综艺,由于媒体命名等的一些问题,大家都在转投以tmdb为主,只支持tvdb的sonarr无法很好地刮削并管理整个媒体库

### 参考资料

![image](https://github.com/jxxghp/MoviePilot/assets/47959853/80100572-ce16-4f72-91bd-5d038b890b9f)
![image](https://github.com/jxxghp/MoviePilot/assets/47959853/d1bc060a-dd49-45cb-857a-9e3d76a20e13)

**Comment by Linyue-GitHub** [comment]
顶
